albumentations/augmentations/transforms.py

```diff
@@ -3320,3 +3322,73 @@ def get_transform_init_args_names(self):
             "mask_fill_value",
             "random_offset",
         )
+
+
+class Defocus(ImageOnlyTransform):
+    """
+    Apply defocus transform. See arXiv:1903.12261 [cs.LG].
+
+    Args:
+        severity (int, (int, int)): maximum severity for defocusing the input image.
+            Should be in range [1, 5]. Default: [1, 2].
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+
+    Image types:
+        Any
+    """
+
+    def __init__(self, severity=3, always_apply=False, p=0.5):
+        super(Defocus, self).__init__(always_apply, p)
+        self.severity = to_tuple(severity, low=1)
+
+        assert self.severity[0] >= 1, "Expected minimal severity greater than or equal to 1, got {}".format(
```

Comment:
Can you please change assertion to raising a run-time error in case incorrect range of argument? As a rule of thumb - asserts for tests, exceptions for the run-time errors.